static/src/javascripts/projects/common/utils/config.js

```diff
@@ -19,6 +19,10 @@ define([
         config.page.adUnit = ['/', config.page.dfpAccountId, '/', adUnitOverride].join('');
     }
 
+    config.ophan.pageViewId = new Date().getTime().toString(36) + 'xxxxxxxxxxxx'.replace(/x/g, function() {
+                                    return Math.floor(Math.random() * 36).toString(36);
+                                  });
+
```

Comment:
Can you comment to cross link to ophan's `transmit.coffee`? A reader of this code needs to be completely clear that this logic is fundamental to the operation of ophan and should not be "optimizied" or "refactored" without consideration of the wider impact.